Dockerfile.wormchain

```diff
@@ -13,6 +13,7 @@ COPY ./sdk /sdk
 RUN go mod download
 
 COPY ./wormchain .
+RUN rm -rf ./build/config/gentx/
```

Comment:
Rather than removing the directory here can we add it to the `.dockerignore` file?